### PR TITLE
:seedling: Update Controller Runtime to v0.7.0-alpha.8

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
Updates controller runtime to the latest alpha version

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ issues:
   exclude:
     - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
     - "G108: Profiling endpoint is automatically exposed on /debug/pprof"
+    - "fake.NewFakeClientWithScheme is deprecated: Please use NewClientBuilder instead."
+    - "fake.NewFakeClient is deprecated: Please use NewClientBuilder instead."
```
Comment (vincepri) [inline .golangci.yml:28]: I'll open an issue for this one, so it can be picked up by a contributor

**Comment by fabriziopandini** [comment]
lgtm pending fix to test failures (at least one does not seem a flake)
also, should we document in the v1alpha3 to v1alpha4 changes the removal of the util func ?

**Comment by vincepri** [comment]
@fabriziopandini Yeah, was just waiting for 3976 to be merged first

**Comment by vincepri** [comment]
@fabriziopandini This should be good to go

**Pull request merged**